test/contracts/attestations.ts

```diff
@@ -2,7 +2,7 @@ import { TransactionReceipt } from '@ethersproject/abstract-provider'
 import { BigNumber, Wallet } from 'ethers'
```

Comment:
I think `test/contracts/` would be better named `test/wormhole/` as it contains helpers that are specific to the wormhole bridge (with the exception of `dai.ts` but I think that's ok)